xprof/btx_timeline.cpp

```diff
@@ -5,7 +5,7 @@
 #include <iomanip>  // set precision
 #include <iostream> // stdcout
 #include <map>
-#include <optional>
+#include <string>
```

Comment:
but you are still using optonal

Reply:
It is already included in xprof_utils.hpp.